locations/spiders/ocharleys.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+import json
+import re
+
+import scrapy
+
+from locations.hours import OpeningHours
+from locations.items import Feature
+
+
+class OcharleysSpider(scrapy.Spider):
+    name = "ocharleys"
```

Comment:
Can you please change the spider name here, class name, and file name to reference that it's US-only?

Reply:
Yes, fixed